### PR TITLE
Fix/claim modal

### PR DESCRIPTION
**Pull request by lucca65**
## What issue does this PR close
None, but we had reports from users not being able to claim nothing.

## Changes Proposed ( a list of new changes introduced by this PR)
Add back the claim modal into the community page. I've also started using the new modal pattern, much better!

## How to test ( a list of instructions on how to test this PR)
Try to claim an action on the community page!

### CODE REVIEW

File: src/elm/Session/LoggedIn.elm [hunk 1]
```diff
@@ -160,7 +160,7 @@ initModel shared authModel accountName selectedCommunity =
     , auth = authModel
     , feedback = Hidden
     , showCommunitySelector = False
-    , hasShop = False
```
Comment (lucca65) [inline src/elm/Session/LoggedIn.elm:163]: I've added this from the other PR #330 so I could do a deploy for MUDA (no one is being able to claim actions!!)

File: src/elm/Page/Community.elm [hunk 2]
```diff
@@ -482,6 +484,51 @@ viewAction loggedIn metadata maybeDate action =
...
+                            , if not isLoading then
+                                onClick CloseClaimConfirmation
+
+                              else
+                                onClick NoOp
+                            , disabled isLoading
+                            ]
+                            [ text_ "dashboard.check_claim.no" ]
+                        , button
+                            [ class "flex-1 block button button-primary button-lg w-full md:w-40"
```
Comment (amiskov) [inline src/elm/Page/Community.elm:515]: There are predefined classes for this kind of buttons from the modal's footer: `modal-cancel` and `modal-accept`. Please, see [`src/elm/Page/Dashboard.elm:225`](https://github.com/cambiatus/frontend/blob/master/src/elm/Page/Dashboard.elm#L225) for the reference (the claim popup on the Dashboard).

File: src/elm/Page/Community.elm [hunk 3]
```diff
@@ -482,6 +484,51 @@ viewAction loggedIn metadata maybeDate action =
             ]
 
 
+viewClaimModal : LoggedIn.Model -> Model -> Html Msg
+viewClaimModal loggedIn model =
+    case model.modalStatus of
+        Opened isLoading actionId ->
```
Comment (RodrigoLeiteF) [inline src/elm/Page/Community.elm:490]: Just a small thing. This should be `open` instead. `Opened` is used as the past tense of `open` (he opened the door). We use `open` when we want to denote `not closed`.

File: src/elm/Page/Community.elm (hunk 2 above)
Comment (lucca65) [reply src/elm/Page/Community.elm:515]: Nice! thanks for the review and pointing out an example! It helped a lot 💪

File: src/elm/Page/Community.elm (hunk 3 above)
Comment (lucca65) [reply src/elm/Page/Community.elm:490]: nice! I'll fix it! small change but important

**Pull request merged**